flask_consulate/consul.py

```diff
@@ -40,11 +40,17 @@ def __init__(self, app=None, **kwargs):
             os.environ.get('CONSUL_PORT', 8500)
         self.max_tries = self.kwargs.get('max_tries', 3)
 
-        self.session = None
+        self.consul = None
+        self._session = None # for backward compatibility
 
         if app is not None:
             self.init_app(app)
 
+    @property
+    def session(self):
+        self.app.logger.warning("session field deprecated - please use .consul")
```

Comment:
I'm cool with this, and here's an alternative just in case you weren't aware: https://docs.python.org/3/library/warnings.html#warning-categories

Reply:
thanks. I did give warnings.warn a try - but the output is captured by the flask framework in the same way as print() etc. So I figured people won't actually see the warning output. Unless I'm missing something.